lib/plugins/plugins.rb

```diff
@@ -32,8 +32,10 @@ def load
     file_location = @settings.plugins['location'] ? @settings.plugins['location'] : '../../config/plugins/'
 
     yaml_files = File.expand_path(file_location + '*.yml', File.dirname(__FILE__))
+    dynamic_port = 48130
```

Comment:
Can log a separate issue, but I feel like there should be a check here if these ports are already taken by something else (potentially even a previously loaded container).